### PR TITLE
Add support for urls from env vars

### PR DESCRIPTION
**Pull request by ryansch**
There's a bit of extra plumbing in this PR to add a lot of our docker stuff.

This is mostly about allowing this:
```
Aide.configure do |config|  
  config.add_service name: :mysql, url_env_key: "AIDE_DATABASE_URL"
end
```

With that I can override the url with that env var. When I later call `Aide.service(:mysql).host`, it'll give me back either the parsed host from the env var or the address of the service in consul.

### CODE REVIEW

File: docker/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+su-exec "${FIXUID:?Missing FIXUID var}:${FIXGID:?Missing FIXGID var}" fixuid
+
+"${WORKSPACE_DIR}/docker/chown-dirs.sh"
+set +e
+su-exec deploy touch /home/deploy/.config/irb/irb_history
+su-exec deploy ln -s /home/deploy/.config/irb/irb_history /home/deploy/.irb_history
+set -e
+
+install_tls_certs() {
+  if [ -n "$(ls -A /usr/local/share/ca-certificates)" ]; then
```
Comment (morganick) [inline docker/entrypoint.sh:14]: What certs are here?

File: docker/irbrc [hunk 2]
```diff
@@ -0,0 +1,9 @@
+IRB.conf[:SAVE_HISTORY] = 1000
+IRB.conf[:USE_AUTOCOMPLETE] = false
```
Comment (morganick) [inline docker/irbrc:2]: Dang straight

File: docker/entrypoint.sh (hunk 1 above)
Comment (ryansch) [reply docker/entrypoint.sh:14]: This is standard boilerplate that we use to inject our dev CA.

**Review by morganick — APPROVED** [review]
LGTM 👍

**Pull request merged**